apps/menu-bar/modules/web-authentication-session/ios/WebAuthenticationSessionModule.swift

```diff
@@ -0,0 +1,35 @@
+import ExpoModulesCore
+import AuthenticationServices
+
+public class WebAuthenticationSessionModule: Module {
+  var authContextProvider = WebAuthContextProvider()
+  var authSession: ASWebAuthenticationSession?
+
+  public func definition() -> ModuleDefinition {
+    Name("WebAuthenticationSession")
+
+    AsyncFunction("openAuthSessionAsync") { (urlString: String, promise: Promise) in
+      guard let url = URL(string: urlString) else {
+        promise.reject("INVALID_URL", "Invalid URL provided")
+        return
+      }
+
+      authSession = ASWebAuthenticationSession(url: url, callbackURLScheme: "expo-orbit") { callbackURL, error in
+        if let error = error {
```

Comment:
```suggestion
        if let error {
```
If they have the same name we can omit the `= *`